dev-java/dynalang/dynalang-0.4-r1.ebuild

```diff
@@ -10,7 +10,7 @@ inherit java-pkg-2 java-ant-2
 
 MY_P="${PN}-mop-${PV}"
 
-DESCRIPTION="Library for implementation of interoperable metaobject protocols for dynamic languages"
+DESCRIPTION="Implementation of interoperable metaobject protocols for dynamic languages"
```

Comment:
These descriptions aren't equivalent. I'm not sure what I'd put instead but I think "library" needs to be in there somewhere.